src/Uno.UI/UI/Xaml/Controls/PasswordBox/PasswordBox.cs

```diff
@@ -17,6 +17,9 @@ public partial class PasswordBox : TextBox
 		private readonly SerialDisposable _revealButtonSubscription = new SerialDisposable();
 
 		public PasswordBox()
+#if __MACOS__
+			:  base(true)
```

Comment:
it does not look like this TextBox is used in any platform. You can remove it from TextBox completely.